### ISSUE TITLE
Avoid repetition

### ISSUE THREAD

**Issue by Rodrigap** [almaforrer/prj-rev-bwfs-dasmoto#3]
https://github.com/almaforrer/prj-rev-bwfs-dasmoto/blob/afb4663aab75533a8f9c11bceda2ffd8c33dd03d/Dasmotos_ArtsAndCrafts/resources/style.css#L6

I like that you added a universal style for Helvetica, but the whole point of that is to eliminate repetition! You don't need to keep writing the helvetica font line on all of these selectors.

**Comment by almaforrer** [comment]
Ah yes totally! I will correct that in my file! Thank you! 
Also, I'm still wondering if there is a way to color one word in a sentence with something else than <span>  ?  Thank you very much for your comments! I'm very happy I did it! 
Hope I'll understand Terminal for mac as well as I understood html! hehe ..Have a nice day! Alma

**Comment by Rodrigap** [comment]
Yeah, you can color just one word, you just have to separate it into its own h2! Your HTML would look like this:

`<h2> This is a white sentence, but <h2 class='blue'> THIS </h2> word is blue </h2>`

And then your CSS would look like:

```
h2 {
color: white
}

.blue {
color: blue
}
```

This way, the word with the 'blue' class is turned blue.
